### PR TITLE
Embed beam width in model and make parameter optional in API

### PR DESCRIPTION
**Pull request by reuben**

### CODE REVIEW

**Comment by reuben** [comment]
@lissyx this is the same as the other PR except for these Windows fixes: https://github.com/mozilla/DeepSpeech/compare/c1e0ec2e448fd722a131ff827fecf5ecc405b2c0..3637f88c065588ab03ac91c483de8cbf2d092c57

File: evaluate_tflite.py [hunk 1]
```diff
@@ -36,7 +36,8 @@
 
 def tflite_worker(model, scorer, queue_in, queue_out, gpu_mask):
     os.environ['CUDA_VISIBLE_DEVICES'] = str(gpu_mask)
-    ds = Model(model, BEAM_WIDTH)
+    ds = Model(model)
+    ds.setBeamWidth(BEAM_WIDTH)
```
Comment (lissyx) [inline evaluate_tflite.py]: So we keep forcing hard-coded values (potentially) different from what the model has ? I don't feel safe with that :/

Comment (kdavis-mozilla) [reply evaluate_tflite.py]: @lissyx Why?

Comment (lissyx) [reply evaluate_tflite.py]: We force-overwrite default model's values by some random magic constant. That was fine when we had no default value recorded with the model, but now it's just asking for people to make mistake and the complain :)

**Comment by reuben** [comment]
Forgot about that. Test with the change: https://community-tc.services.mozilla.com/tasks/ZHnaHlXFRiGRe6E6b_2MtA

**Pull request closed without merging**